test/DynamoCoreWpfTests/WorkspaceOpeningTests.cs

```diff
@@ -151,15 +151,16 @@ public void ManipulatingWorkspaceDoesNotAffectZoom()
         private WorkspaceModel OpenWorkspaceFromSampleFile()
         {
             var examplePath = Path.Combine(SampleDirectory, @"en-US\Basics\Basics_Basic01.dyn");
-            ViewModel.Model.OpenFileFromPath(examplePath);
+            ViewModel.OpenCommand.Execute(examplePath);
```

Comment:
The view block was coming back empty using the previous opening strategy causing a series of tests to fail